tests/e2e-test.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { Client } from "ts-postgres";
+import { rootConnect } from "./postgres-utils";
+import request from 'supertest';
+import { POSTGRES, SUPERSET, DBT_POSTGRES } from "../scripts/config";
+
+describe("Main workflow Test Suite", () => {
+  let client: Client;
+
+  beforeAll(async () => {
+    client = await rootConnect();
+  });
+
+  afterAll(async () => {
+    await client.end();
+  });
```

Comment:
Maybe these 2 could be one liner, as in b`eforeAll(async () => client = await rootConnect());` and
  `afterAll(async () => await client.end());`
  but that's pretty trivial I think.